app/views/catalog/_show_main_content.html.erb

```diff
@@ -1,17 +1,23 @@
-<% @page_title = t('blacklight.search.show.title', :document_title => document_show_html_title, :application_name => application_name).html_safe %>
+<% @page_title = t('blacklight.search.show.title', document_title: Deprecation.silence(Blacklight::BlacklightHelperBehavior) { document_show_html_title }, application_name: application_name).html_safe %>
```

Comment:
Is there a ticket we need to go along with this deprecation silencing?

Reply:
No, this is just me copying over the new `show_main_content` from blacklight.